packages/caip/src/adapters/coincap/generated/eip155_1/adapter.json

```diff
@@ -1 +1 @@
/erc20:0xa017ac5fac5941f95010b12570b812c974469c2c":"proxeus","eip155:1/erc20:0x55648de19836338549130b1af587f16bea46f66b":"publica","eip155:1/erc20:0x846c66cf71c43f80403b51fe3906b3599d63336f":"pumapay","eip155:1/erc20:0xa15c7ebe1f07caf6bff097d8a589fb8ac49ae5b3":"pundi-x","eip155:1/erc20:0x9196e18bc349b1f64bc08784eae259525329a1ad":"pussy-financial","eip155:1/erc20:0x7703c35cffdc5cda8d27aa3df2f9ba6964544b6e":"pylon-network","eip155:1/erc20:0x618e75ac90b12c6049ba3b27f5d5f8651b0037f6":"qash","eip155:1/erc20:0x4234f63b1d202f6c016ca3b6a0d41d7d85f17716":"quanta-utility-token","eip155:1/erc20:0xf9e5af7b42d31d51677c75bbbd37c1986ec79aee":"quickx-protocol","eip155:1/erc20:0x264dc2dedcdcbb897561a57cba5085ca416fb7b4":"qunqun","eip155:1/erc20:0x076a93a40bf9e0d21d3f75dd1e0584ddbe0f9d1a":"qurito","eip155:1/erc20:0x1183f92a5624d68e85ffb9170f16bf0443b4c242":"qvolta","eip155:1/erc20:0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6":"raiden-network-token","eip155:1/erc20:0xeb953eda0dc65e3246f43dc8fa13f35623bdd5ed":"rainicorn","eip155:1/erc20:0x6e5a43db10b04701385a34afb670e404bc7ea597":"rakon","eip155:1/erc20:0x33d0568941c0c64ff7e0fb4fba0b11bd37deed9f":"ramp","eip155:1/erc20:0xd291e7a03283640fdc51b121ac401383a46cc623":"rari-governance-token","eip155:1/erc20:0x436f0f3a982074c4a05084485d421466a994fe53":"rate3","eip155:1/erc20:0x50de6856358cc35f3a9a57eaaa34bd4cb707d2cd":"razor-network","eip155:1/erc20:0x168296bb09e24a88805cb9c33356536b980d3fc5":"rchain","eip155:1/erc20:0x13d0bf45e5f319fa0b58900807049f23cae7c40d":"read","eip155:1/erc20:0x13f25cd52b21650caa8225c9942337d914c9b030":"realchain","eip155:1/erc20:0x1fc5ef0337aea85c5f9198853a6e3a579a7a6987":"reapchain","eip155:1/erc20:0x5f53f7a8075614b699baad0bc2c899f4bad8fbbf":"rebl","eip155:1/erc20:0xe5a3229ccb22b6484594973a03a3851dcd948756":"receive-access-ecosystem","eip155:1/erc20:0x8b39b70e39aa811b69365398e0aace9bee238aeb":"polkafoundry","eip155:1/erc20:0x4a42d2c580f83dce404acad18dab26db11a1750e":"relex","eip155:1/erc20:0x83984d6142934bb535793a82adb0a46ef0f66b6d":"remme","eip155:1/erc20:0xb92f51ce4045212eef8008c2f665da713035267b":"rentledger","eip155:1/erc20:0x5218e472cfcfe0b64a064f055b43b4cdc9efd3a6":"unfederalreserve","eip155:1/erc20:0x6425c6be902d692ae2db752b3c268afadb099d3b":"restart-energy-mwat","eip155:1/erc20:0x48f775efbe4f5ece6e0df2f7b5932df56823b990":"revain","eip155:1/erc20:0x17ef75aa22dd5f6c2763b8304ab24f40ee54d48a":"revolution-populi","eip155:1/erc20:0x3ffffa8f3cc943e43f9f17a83cbb18f4bbb9f4ac":"rewardiqa","eip155:1/erc20:0xa1d6df714f91debf4e0802a542e13067f31b8262":"redfox-labs","eip155:1/erc20:0xb24754be79281553dc1adc160ddf5cd9b74361a4":"rialto","eip155:1/erc20:0x6123b0049f904d730db3c36a31167d9d4121fa6b":"ribbon-finance","eip155:1/erc20:0x8d5682941ce456900b12d47ac06a88b47c764ce1":"rightmesh","eip155:1/erc20:0x7f86c782ec802ac402e0369d2e6d500256f7abc5":"ring-x-platform","eip155:1/erc20:0xf970b8e36e23f7fc3fd752eea86f8be8d83375a6":"ripio-credit-network","eip155:1/erc20:0x3d1ba9be9f66b8ee101911bc36d3fb562eac2244":"rivetz"}
```

Comment:
Can't comment on files with new GH CR preview (unless it's a skill issue?) but ran a regen since the generated asset data still had RUJI icon on `feat-ruji` and needed a regen (thankfully, develop still has RUJI PR commit in, so we *can* do that).
Failure to do that means icon would work while this branch is there, and get rekt once it gets deleted (which it did during ops testing)